auto_nag/scripts/severity_inconsistency.py

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+
+class SeverityInconsistency(BzCleaner):
+    def description(self):
+        return "Bugs with inconsistent severity flags"
+
+    def has_needinfo(self):
+        return True
+
+    def handle_bug(self, bug, data):
+        self.extra_ni = data
+
+        bugid = str(bug["id"])
+
+        def is_security_flag(flag):
+            return flag.startswith("[access-s")
+
+        flags = bug["whiteboard"].split(", ")
+        whiteboard_severity = next(filter(is_security_flag, flags), None)
```

Comment:
Nit: we can do:
```Python
whiteboard_severities = [flag for flag in flags if flag.startswith("[access-s")]
assert len(whiteboard_severities) == 1
whiteboard_severity = whiteboard_severities[0]
```

This way we ensure there is one and only one "access-sX" severity for each bug.